src/pages/iou/MoneyRequestMerchantPage.js

```diff
@@ -53,6 +53,7 @@ function MoneyRequestMerchantPage({iou, route}) {
     const {inputCallbackRef} = useAutoFocusInput();
     const iouType = lodashGet(route, 'params.iouType', '');
     const reportID = lodashGet(route, 'params.reportID', '');
+    const isEmptyMerchant = iou.merchant === '' || iou.merchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT || iou.merchant === CONST.TRANSACTION.DEFAULT_MERCHANT;
```

Comment:
Same as above `Request` is not empty merchant I think
```suggestion
    const isEmptyMerchant = iou.merchant === '' || iou.merchant === CONST.TRANSACTION.PARTIAL_TRANSACTION_MERCHANT;
```